blueutil.m

```diff
@@ -51,6 +53,119 @@ int assert_reg(int errcode, const regex_t *restrict preg, char *reason) {
 
 void _NSSetLogCStringFunction(void (*)(const char *, unsigned, BOOL));
 
+// Mock IOBluetoothDevice for subprocess method
+@interface MockBluetoothDevice : NSObject
+@property (nonatomic, strong, readonly) NSString *address;
+@property (nonatomic, strong, readonly) NSString *name;
+@property (nonatomic, assign, readonly) BOOL paired;
+@property (nonatomic, assign, readonly) BOOL connected;
+@property (nonatomic, assign, readonly) BOOL favorite;
+@property (nonatomic, strong, readonly) NSDate *recentAccessDate;
+@property (nonatomic, assign, readonly) int rssi;
```

Comment:
Why not use char here?

Reply:
Force of habit.

I'll change it to `char`.

Reply:
Done